docs/resources/object_bucket.md

```diff
@@ -21,6 +21,85 @@ resource "scaleway_object_bucket" "some_bucket" {
 }
 ```
 
+### Using object lifecycle
+
+```hcl
+resource "scaleway_object_bucket" "main"{
+  name = "mybuckectid"
+  region = "fr-par"
+  acl = "private"
+  
+  lifecycle_rule {
+      id      = "id1"
+      prefix  = "path1/"
+      enabled = true
+  
+      expiration {
+        days = 365
+      }
+  
+      transition {
+        days          = 120
+        storage_class = "GLACIER"
+      }
+  }
+  
+  lifecycle_rule {
+      id      = "id2"
+      prefix  = "path2/"
+      enabled = true
+  
+      expiration {
+        days = "50"
+      }
+  }
+  
+  lifecycle_rule {
+      id      = "id3"
+      prefix  = "path3/"
+      enabled = true
+  
+      tags = {
+        "tagKey"    = "tagValue"
+        "terraform" = "hashicorp"
+      }
+  
+      expiration {
+        days = "1"
+      }
+  }
+  
+  lifecycle_rule {
+      id      = "id4"
+      enabled = true
+  
+      tags = {
+        "tagKey"    = "tagValue"
+        "terraform" = "hashicorp"
+      }
+  
+      transition {
+        days          = 0
+        storage_class = "GLACIER"
+      }
+  }
+  
+  lifecycle_rule {
```

Comment:
What is the difference between this example and the one above? Is it really necessary?

Reply:
I've modify the use case